Lib/test/test_buffer.py

```diff
@@ -89,7 +89,8 @@
     'i':(-(1<<31), 1<<31), 'I':(0, 1<<32),
     'l':(-(1<<31), 1<<31), 'L':(0, 1<<32),
     'q':(-(1<<63), 1<<63), 'Q':(0, 1<<64),
-    'f':(-(1<<63), 1<<63), 'd':(-(1<<1023), 1<<1023)
+    'e':(-65519, 65520),   'f':(-(1<<63), 1<<63),
```

Comment:
I know it's only test code, but this doesn't look quite right; why `-65519` rather than `-65520`?

Reply:
Ah, sorry, I see: the least (i.e., most negative) _representable value is `-65504`; `-65520` would round _down_ to infinity.

Reply:
Because:
```python
>>> np.array([65519, -65519], dtype='e')
array([ 65504., -65504.], dtype=float16)
>>> np.array([65520, -65520], dtype='e')
array([ inf, -inf], dtype=float16)
```